flight/targets/brainre1/fw/pios_board.c

```diff
@@ -439,57 +460,79 @@ void PIOS_Board_Init(void) {
 			0,                                   // dsm_mode
 			NULL);                               // sbus_rcvr_cfg
 
-	/* Multi-function port (pwm, serial, etc) */
+	/* Multi-function port (pwm, 1 serial, 2 serial) */
 	uint8_t hw_mp;
 	HwBrainRE1MultiPortGet(&hw_mp);
 
+	/* Configure PWM Outputs */
+#if defined(PIOS_INCLUDE_SERVO) && defined(PIOS_INCLUDE_TIM)
 	switch (hw_mp) {
-		case HWBRAINRE1_MULTIPORT_PWM:
-			// for us, this means PWM output, not input
+		case HWBRAINRE1_MULTIPORT_PWMOUT4:
+			PIOS_Servo_Init(&pios_servo_4_cfg);
+			break;
+		case HWBRAINRE1_MULTIPORT_PWMOUT6:
+			PIOS_Servo_Init(&pios_servo_6_cfg);
+			break;
+		case HWBRAINRE1_MULTIPORT_PWMOUT8:
+			PIOS_Servo_Init(&pios_servo_8_cfg);
 			break;
 		default:
-			PIOS_HAL_ConfigurePort(hw_mp,                // port type protocol
-					&pios_usart6_cfg,                    // usart_port_cfg
-					&pios_usart_com_driver,              // com_driver
-					NULL,                                // i2c_id
-					NULL,                                // i2c_cfg
-					NULL,                                // ppm_cfg
-					NULL,                                // pwm_cfg
-					PIOS_LED_ALARM,                      // led_id
-					NULL,                                // dsm_cfg
-					0,                                   // dsm_mode
-					NULL);                               // sbus_rcvr_cfg
+			PIOS_Servo_Init(&pios_servo_4_cfg);
 			break;
 	}
+#endif /* defined(PIOS_INCLUDE_SERVO) && defined(PIOS_INCLUDE_TIM) */
 
-	// Enable / disable RE1 specific hardware inverters and FrSky RSSI support
-	switch (hw_mp) {
-		case HWBRAINRE1_MULTIPORT_FRSKYSENSORHUB:
-			PIOS_RE1FPGA_MPTxPinMode(false, true);
+	if (hw_mp == HWBRAINRE1_MULTIPORT_PWMOUT4) {
```

Comment:
I don't think we should offer this option. UART4 is on the on the first two motor outputs and this makes it very confusing for users..